Theme1/src/firsttheme/SudokuCheck.java

```diff
@@ -0,0 +1,71 @@
+package firsttheme;
+
+public class SudokuCheck {
+    public static boolean sudokuChecker(int[][] sudoku) {
+        int[] tmp = new int[9];
+        if (sudoku.length != 9) throw new IllegalArgumentException("wrong size of sudoku!");
+        for(int[] sudokuRow : sudoku) {
+            if(sudokuRow.length != 9) throw new IllegalArgumentException("wrong size of sudoku");
+        }
+        //check of row
+        for(int[] line : sudoku)
+            if (!lineChecker(line)) return false;
```

Comment:
As I mentioned before, it's a good practice to avoid short form of operators. Example:
```
for(int[] line : sudoku) {
  if (!lineChecker(line)) {
    return false;
  }
}
```